### PR TITLE
EFRS-969. Add an ability to handle images in BASE64 format CORE

### PR DESCRIPTION
**Pull request by azolotnitsky**

### CODE REVIEW

**Comment by azolotnitsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by azolotnitsky** [comment]
recheckcla

**Comment by azolotnitsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by azolotnitsky** [comment]
recheckcla

File: embedding-calculator/src/docs/find_faces_base64_post.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+tags:
+  - Core
+summary: 'Find faces in the given image in BASE64 format and return their bounding boxes.'
+description: 'Returns bounding boxes of detected faces on the image.'
+operationId: findFacesBase64Post
+consumes:
+  - multipart/form-data
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_base64_post.yml:7]: It definitely should consume `application/json`
Please also check if this swagger is working. To check, just open:
http://localhost:3300/apidocs
in browser, find the needed rest endpoint and try to invoke it using swagger

File: embedding-calculator/src/docs/find_faces_base64_post.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+produces:
+  - application/json
+parameters:
+  - in: formData
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_base64_post.yml:11]: I believe it should be `body`, not `formData`

File: embedding-calculator/src/docs/find_faces_base64_post.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    name: file
+    type: file
+    required: 'true'
+    description: 'A picture with at least one face in BASE64 format.'
+  - in: formData
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_base64_post.yml:16]: I believe it should be `query`, not `formData`
And yes, it looks like some parameters in `find_faces_post.yml` also defined wrong. I would be glad if you fix there too

**Comment by azolotnitsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by azolotnitsky** [comment]
recheckcla

File: embedding-calculator/src/docs/find_faces_base64_post.yml (hunk 1 above)
Comment (azolotnitsky) [reply embedding-calculator/src/docs/find_faces_base64_post.yml]: Checked. It is working

File: embedding-calculator/src/docs/find_faces_base64_post.yml (hunk 2 above)
Comment (azolotnitsky) [reply embedding-calculator/src/docs/find_faces_base64_post.yml]: Fixed

File: embedding-calculator/src/docs/find_faces_base64_post.yml (hunk 3 above)
Comment (azolotnitsky) [reply embedding-calculator/src/docs/find_faces_base64_post.yml]: Fixed

**Comment by azolotnitsky** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by azolotnitsky** [comment]
recheckcla

File: embedding-calculator/src/docs/find_faces_base64_post.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    name: file
+    type: file
+    required: 'true'
+    description: 'A picture with at least one face in BASE64 format.'
+  - in: query
+    name: limit
+    description: 'The limit of faces that you want recognized. Value of 0 represents no limit.'
+    type: integer
+    default: 0
+  - in: formData
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_base64_post.yml:21]: This is also `query`, the same in `find_faces_post.yml`

File: embedding-calculator/src/docs/find_faces_base64_post.yml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    name: det_prob_threshold
+    description: 'The minimum required confidence that a found face is actually a face. Decrease this value if faces are not detected. Valid values are in the range (0;1).'
+    type: float
+  - in: formData
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_base64_post.yml:25]: This is also `query`, the same in `find_faces_post.yml`

File: embedding-calculator/src/docs/find_faces_post.yml [hunk 6]
```diff
@@ -4,16 +4,16 @@ summary: 'Find faces in the given image and return their bounding boxes.'
 description: 'Returns bounding boxes of detected faces on the image.'
 operationId: findFacesPost
 consumes:
-  - multipart/form-data
+  - application/json
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_post.yml:7]: Please, revert it, it should be `multipart/form-data`

File: embedding-calculator/src/docs/find_faces_post.yml [hunk 7]
```diff
@@ -4,16 +4,16 @@ summary: 'Find faces in the given image and return their bounding boxes.'
...
 produces:
   - application/json
 parameters:
-  - in: formData
+  - in: body
```
Comment (pospielov) [inline embedding-calculator/src/docs/find_faces_post.yml:11]: Please, revert it, it should be `formData`

**Review by pospielov — APPROVED** [review]

**Comment by azolotnitckii-exadel** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by azolotnitckii-exadel** [comment]
recheckcla

File: embedding-calculator/src/docs/find_faces_base64_post.yml (hunk 4 above)
Comment (azolotnitckii-exadel) [reply embedding-calculator/src/docs/find_faces_base64_post.yml]: Fixed

File: embedding-calculator/src/docs/find_faces_base64_post.yml (hunk 5 above)
Comment (azolotnitckii-exadel) [reply embedding-calculator/src/docs/find_faces_base64_post.yml]: Fixed

File: embedding-calculator/src/docs/find_faces_post.yml (hunk 6 above)
Comment (azolotnitckii-exadel) [reply embedding-calculator/src/docs/find_faces_post.yml]: Fixed

File: embedding-calculator/src/docs/find_faces_post.yml (hunk 7 above)
Comment (azolotnitckii-exadel) [reply embedding-calculator/src/docs/find_faces_post.yml]: Fixed

**Pull request merged**